Resources/Maps/_NF/Shuttles/akupara.yml

```diff
@@ -0,0 +1,4264 @@
+      pos: -4.5,2.5
+      parent: 1
+  - uid: 347
+    components:
+    - type: Transform
+      rot: 3.141592653589793 rad
+      pos: -4.5,-1.5
+      parent: 1
+- proto: IntercomCommon
+  entities:
+  - uid: 348
+    components:
+    - type: Transform
+      rot: -1.5707963267948966 rad
+      pos: 5.5,11.5
+      parent: 1
+- proto: Jug
+  entities:
+  - uid: 349
+    components:
+    - type: Transform
+      pos: 3.3617766,8.23579
+      parent: 1
+  - uid: 350
+    components:
+    - type: Transform
+      pos: 3.5805264,7.9857903
+      parent: 1
+- proto: KitchenReagentGrinder
+  entities:
+  - uid: 351
+    components:
+    - type: Transform
+      pos: 3.5,8.5
+      parent: 1
+- proto: LockerAtmosphericsFilledHardsuit
+  entities:
+  - uid: 352
+    components:
+    - type: Transform
+      pos: 5.5,-2.5
+      parent: 1
+- proto: LockerBotanistFilled
+  entities:
+  - uid: 353
+    components:
+    - type: Transform
+      pos: 1.5,-4.5
+      parent: 1
+- proto: LockerCaptain
+  entities:
+  - uid: 354
+    components:
+    - type: Transform
+      pos: 3.5,11.5
+      parent: 1
+    - type: Lock
+      locked: False
+    - type: Fixtures
+      fixtures:
+        fix1:
+          shape: !type:PolygonShape
+            radius: 0.01
+            vertices:
+            - -0.25,-0.48
+            - 0.25,-0.48
+            - 0.25,0.48
+            - -0.25,0.48
+          mask:
+          - Impassable
+          - TableLayer
+          - LowImpassable
+          layer:
+          - BulletImpassable
+          - Opaque
+          density: 75
+          hard: True
+          restitution: 0
+          friction: 0.4
+    - type: EntityStorage
+      open: True
+      removedMasks: 20
+    - type: PlaceableSurface
+      isPlaceable: True
+- proto: LockerChemistryFilled
+  entities:
+  - uid: 355
+    components:
+    - type: Transform
+      pos: 5.5,9.5
+      parent: 1
+- proto: MaintenancePlantSpawner
+  entities:
+  - uid: 356
+    components:
+    - type: Transform
+      pos: -2.5,-2.5
+      parent: 1
+- proto: MaterialReclaimer
+  entities:
+  - uid: 357
+    components:
+    - type: Transform
+      pos: 6.5,-7.5
+      parent: 1
+- proto: MopBucketFull
+  entities:
+  - uid: 358
+    components:
+    - type: Transform
+      pos: 2.5,-9.5
+      parent: 1
+- proto: MopItem
+  entities:
+  - uid: 359
+    components:
+    - type: Transform
+      pos: 2.5,-10.5
+      parent: 1
+- proto: NitrogenCanister
+  entities:
+  - uid: 360
+    components:
+    - type: Transform
+      anchored: True
+      pos: 1.5,2.5
+      parent: 1
+    - type: Physics
+      bodyType: Static
+- proto: OxygenCanister
+  entities:
+  - uid: 361
+    components:
+    - type: Transform
+      anchored: True
+      pos: 2.5,3.5
+      parent: 1
+    - type: Physics
+      bodyType: Static
+- proto: PestSpray
+  entities:
+  - uid: 362
+    components:
+    - type: Transform
+      pos: -0.5,2.5
+      parent: 1
+- proto: PortableGeneratorSuperPacmanShuttle
```

Comment:
This generator getting rads added at some point
Either make it fully closed in a rads protection chamber with full locker kit for rads, or replace with plasma generators

Also use shuttle generators.

Reply:
Double checked generator type used, confirmed to be the shuttle variant. Generator enclosure adjusted to add radiation shutter on access side.